src/store/SavedQueriesTypes.ts

```diff
@@ -0,0 +1,110 @@
+import { ThunkDispatch } from 'redux-thunk';
+
+import { RootState } from './rootState';
+import { VirtualStudy } from './virtualStudiesTypes';
+
+type FilesQueryContent = {
+  Families: number;
```

Comment:
why do theses properties start with an uppercase letter ?

Reply:
I had the same reaction. I just don't know - that's how the data is received.